templates/angular/igx-ts/custom-templates/tree-grid-finjs/files/src/app/__path__/__name__.component.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { FormsModule } from '@angular/forms';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { $(ClassName)Component } from './tree-grid-fintech.component';
+import { IgxTreeGridModule, IgxDatePickerModule, IgxCheckboxModule } from 'igniteui-angular';
+
+ describe('$(ClassName)Component', () => {
```

Comment:
There's an extra space in front of `describe`, `beforeEach` and `it` and it's making lint angry